orcoursetrion/tests/test_cmd.py

```diff
@@ -49,5 +49,57 @@ def test_cmd_create_xml_repo(self):
                     self.TEST_COURSE,
                     self.TEST_TERM,
                     self.TEST_TEAM,
+                    None,
                     self.TEST_DESCRIPTION
                 )
+        args = [
+            'orcoursetrion', 'create_xml_repo',
+            '-c', self.TEST_COURSE,
+            '-t', self.TEST_TERM,
+            '-d', self.TEST_DESCRIPTION,
+            '-m', 'archlight', 'dreadnought',
```

Comment:
Won't `dreadnought` be parsed as a separate argument here, rather than an answer to `-m`?